docs/bpftool-net.rst

```diff
@@ -34,56 +34,56 @@ NET COMMANDS
-		  Attach bpf program *PROG* to network interface *NAME* with
-		  type specified by *ATTACH_TYPE*. Previously attached bpf program
-		  can be replaced by the command used with **overwrite** option.
-		  Currently, only XDP-related modes are supported for *ATTACH_TYPE*.
-
-		  *ATTACH_TYPE* can be of:
-		  **xdp** - try native XDP and fallback to generic XDP if NIC driver does not support it;
-		  **xdpgeneric** - Generic XDP. runs at generic XDP hook when packet already enters receive path as skb;
-		  **xdpdrv** - Native XDP. runs earliest point in driver's receive path;
-		  **xdpoffload** - Offload XDP. runs directly on NIC on each packet reception;
-
-	**bpftool** **net detach** *ATTACH_TYPE* **dev** *NAME*
-		  Detach bpf program attached to network interface *NAME* with
-		  type specified by *ATTACH_TYPE*. To detach bpf program, same
-		  *ATTACH_TYPE* previously used for attach must be specified.
-		  Currently, only XDP-related modes are supported for *ATTACH_TYPE*.
-
-	**bpftool net help**
-		  Print short help message.
+    bpftool net { show | list } [ dev *NAME* ]
+        List bpf program attachments in the kernel networking subsystem.
+
+        Currently, device driver xdp attachments, tcx and old-style tc
+        classifier/action attachments, flow_dissector as well as netfilter
+        attachments are implemented, i.e., for program types
+        **BPF_PROG_TYPE_XDP**, **BPF_PROG_TYPE_SCHED_CLS**,
+        **BPF_PROG_TYPE_SCHED_ACT**, **BPF_PROG_TYPE_FLOW_DISSECTOR**,
+        **BPF_PROG_TYPE_NETFILTER**.
+
+        For programs attached to a particular cgroup, e.g.,
+        **BPF_PROG_TYPE_CGROUP_SKB**, **BPF_PROG_TYPE_CGROUP_SOCK**,
+        **BPF_PROG_TYPE_SOCK_OPS** and **BPF_PROG_TYPE_CGROUP_SOCK_ADDR**, users
+        can use **bpftool cgroup** to dump cgroup attachments. For sk_{filter,
+        skb, msg, reuseport} and lwt/seg6 bpf programs, users should consult
+        other tools, e.g., iproute2.
+
+        The current output will start with all xdp program attachments, followed
+        by all tcx, then tc class/qdisc bpf program attachments, then
+        flow_dissector and finally netfilter programs. Both xdp programs and
+        tcx/tc programs are ordered based on ifindex number. If multiple bpf
+        programs attached to the same networking device through **tc**, the
+        order will be first all bpf programs attached to tcx, then tc classes,
+        then all bpf programs attached to non clsact qdiscs, and finally all bpf
+        programs attached to root and clsact qdisc.
+
+    bpftool net attach *ATTACH_TYPE* *PROG* dev *NAME* [ overwrite ]
+        Attach bpf program *PROG* to network interface *NAME* with type
+        specified by *ATTACH_TYPE*. Previously attached bpf program can be
+        replaced by the command used with **overwrite** option. Currently, only
+        XDP-related modes are supported for *ATTACH_TYPE*.
+
+        *ATTACH_TYPE* can be of:
```

Comment:
Sorry I missed that one, this list needs to be separated from the rest of the paragraph with a blank line as well:

```suggestion
        *ATTACH_TYPE* can be one of:

```